molecule/default/tests/test_create_and_assign_floating_ip.py

```diff
@@ -0,0 +1,86 @@
+import pytest_rpc as helpers
```

Comment:
I believe this should be
```
import pytest_rpc.helpers as helpers
```

Reply:
I'll update the test using local utils.py in Nova molecule, will add methods of Nova utils.py to pytest-prc in ticket ASC-687